tests/mgbench/runners.py

```diff
@@ -124,7 +125,7 @@ def execute(
         queries=None,
         file_path=None,
         num_workers=1,
-        max_retries: int = 10000,
+        max_retries: int = 50,
```

Comment:
should this stay like this

Reply:
doesn't make sense to me to be such a high number